### PR TITLE
added Soundeffects

### PR DESCRIPTION
**Pull request by ManuShift**
added following soundeffects:

- jumping-sound
- death-sound
- damage-sound

### CODE REVIEW

File: src/arena.py [hunk 1]
```diff
@@ -36,8 +36,8 @@ def set_values_to_pics(cls, pygame, base_path, tile_size):
                     pygame.image.load(base_path + member.filename).convert(), (tile_size, tile_size)
                 )
 
-    blocks_base_path = "../Tiles/"
-    maps_base_path = "./../Maps/"
+    blocks_base_path = "Tiles/"
```
Comment (DerApplaudeur) [inline src/arena.py:39]: This breakes the map/tile loading

**Review by DerApplaudeur — CHANGES_REQUESTED** [review]
every single file link is broken for me.
The sounds are nice but we should add an option to change the volume soon.
You also changed the 2 shots ranged attack into a single shot attack wihtout mentioning it in a commit message or a comment.

File: src/game.py [hunk 2]
```diff
@@ -455,11 +488,11 @@ def player_robot_handling(player_robot):
                     player_robot.melee_attack(pygame, screen, robots, arena)
                     player_robot.melee_cd += 1
                 elif key == pygame.K_r and (player_robot.ranged_cd == 0 or player_robot.ranged_cd == 10):
-                    player_robot.ranged_attack()
```
Comment (DerApplaudeur) [inline src/game.py:458]: This was used for the second shot

**Comment by Luceleven** [comment]
Deprecated

**Pull request closed without merging**